scripts/calendar/hebrew_generator.py

```
@@ -44,49 +44,134 @@
-    h_year_min = g_year_min + hebrew.HEBREW_YEAR_OFFSET
-    h_year_max = g_year_max + hebrew.HEBREW_YEAR_OFFSET + 1
+    h_year_min = g_year_min + HEBREW_YEAR_OFFSET
+    h_year_max = g_year_max + HEBREW_YEAR_OFFSET + 1
 
-    dates = {}
+    dates = defaultdict(dict)
     for h_year in range(h_year_min, h_year_max + 1):
         for h_month, h_day, hol_name in HEBREW_HOLIDAYS:
-            if h_month == 12 and hebrew.leap(h_year):
+            if h_month == 12 and cal.leap(h_year):
                 h_month += 1
```

Ruff (0.15.9)

[warning] `for` loop variable `h_month` overwritten by assignment target

(PLW2901)